<launch>
  <node pkg="lslidar_driver" type="lslidar_driver_node" name="lslidar_driver_node" output="screen">
    <param name="lidar_name" value="M10"/>                                            #雷达选择:M10 M10_P M10_PLUS M10_GPS N10 N10_PLUS L10
    <param name="serial_port" value="/dev/ttyUSB0"/>                                  #雷达连接的串口
    <param name="interface_selection" value="serial"/>                                #接口选择:net 为网口,serial 为串口。
    <param name="frame_id" value="laser_link"/>                                       #激光坐标
    <param name="scan_topic" value="scan"/>                                           #设置激光数据topic名称
    <param name="angle_disable_min" value="0.0"/>                                     #角度裁剪开始值
    <param name="angle_disable_max" value="0.0"/>                                     #角度裁剪结束值
    <param name="min_range" value="0.0"/>                                             #雷达接收距离最小值
    <param name="max_range" value="100.0"/>                                           #雷达接收距离最大值
    <param name="use_gps_ts" value="false"/>                                          #雷达是否使用GPS授时
    <param name="compensation" value="false"/>                                        #雷达是否使用角度补偿功能
    <param name="pubScan" value="true"/>                                              #雷达是否发布scan话题
    <param name="pubPointCloud2" value="false"/>                                      #雷达是否发布pointcloud2话题
    <param name="high_reflection" value="false"/>                                     #M10_P雷达需填写该值,若不确定，请联系技术支持。
    <!--param name="in_file_name" value="$(find lslidar_driver)/pcap/xxx.txt"/-->       #驱动是否使用txt文件读取功能
  </node>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lslidar_driver)/rviz/lslidar.rviz" output="screen"/>      #雷达点云配置
</launch>
